index.js

```diff
@@ -254,13 +254,12 @@ class CerberusClient {
     }
 
     if (!(response.statusCode >= 200 && response.statusCode < 300)) {
-      if (response.headers['content-type'].startsWith('application/json')) {
+      if (response.headers['content-type'] && response.headers['content-type'].startsWith('application/json')) {
```

Comment:
These blocks are the same except for msg content. You could pull most of this outside the block and avoid the repeated code.